hello_snmp_manager/lib/hello_snmp_manager.ex

```diff
@@ -33,7 +33,7 @@ defmodule HelloSnmpManager do
   def handle_info(:tick, state) do
     Logger.debug("Sending SNMP Request for System Information")
 
-    :snmpm.sync_get('default_user', 'default_agent', [
+    :snmpm.sync_get2(~c"default_user", ~c"default_agent", [
```

Comment:
`snmpm:sync_get/3` was removed in OTP25.
ref. https://www.erlang.org/docs/23/general_info/scheduled_for_removal